### PR TITLE
Removes blood boil from blood cult entirely.

### PR DESCRIPTION
**Pull request by Chirynnn**
## About The Pull Request

This removes the blood boil rune from the game entirely.
## Why It's Good For The Game
I think we can all agree that blood cult is quite overpowered in many aspects, and the blood boil rune is the one thing that should definitely get removed. Being able to instantly put 5-6 people who can see the rune into critical state with just THREE cultists is absolutely busted and it should not exist. 

It makes cult more balanced and more reliant on other methods, rather than a relatively easy to make instant crit weapon in a stalemate, for example.
## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image
Gone.
</details>

## Changelog
:cl:
del: Removed the blood boil rune permanently from the game.
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/runes.dm [hunk 1]
```diff
@@ -751,67 +751,6 @@ structure_check() searches for nearby cultist structures required for the invoca
...
-
-/obj/effect/rune/blood_boil/proc/do_area_burn(turf/T, multiplier)
-	set_light(6, 1, color)
-	for(var/mob/living/L in viewers(T))
-		if(!iscultist(L) && L.blood_volume)
-			if(L.anti_magic_check(magic=FALSE,holy=TRUE,major = FALSE))
-				continue
-			L.take_overall_damage(tick_damage*multiplier, tick_damage*multiplier)
-
 //Rite of Spectral Manifestation: Summons a ghost on top of the rune as a cultist human with no items. User must stand on the rune at all times, and takes damage for each summoned ghost.
```
Comment (Penwin0) [inline code/modules/antagonists/cult/runes.dm:755]:
```suggestion

//Rite of Spectral Manifestation: Summons a ghost on top of the rune as a cultist human with no items. User must stand on the rune at all times, and takes damage for each summoned ghost.
```

Spacing you stinky

Comment (Chirynnn) [reply code/modules/antagonists/cult/runes.dm:755]: :tragedy:

**Comment by Oscari-pog** [comment]
sk-

**Comment by PowerfulBacon** [comment]
Couldn't you just lower the damage instead? Reasons for removal state that the damage is too high rather than providing a reason for it to not exist in a non-overpowered state.

**Comment by RDS88-beeman** [comment]
Hey. Feedback. Instead of removing it, replace it with something else. Perhaps rename it to blood chill and replace the damage with a rapid cooling effect, which helps both in finishing off injured, retreating enemies and deterring attackers by slowing them down and providing time to organize a counter-attack or otherwise scatter.

Also. 3:6 participants to murder victims? Lmao low ratio try 1:9 when science applies a maxcap to the summoning or 1:(the entire cult) if you time it with the mass summon. There are more broken things about this game mode than blood boil. 

If you really want to nerf it make the damage apply to the invokers too, make it a sacrifice rather than a sweep.

**Comment by DerpiestBruh** [comment]
ided

**Comment by ivanmixo** [comment]
I can't believe you guys don't see an issue with something that all you need to do to win with is have it in your view. 

Having them freeze would mean they'd be just as good as dead. Lowering the damage won't do anything since it's still free damage (without any danger to yourself might I add) and you can click runes stupid fast. Maybe having it apply damage to you could be okay but then I'm worried it'd fall into the useless category. 

You can't balance something that has it's effects applied in this way. It's either going to be useless or overpowered, so it's better to just have it gone altogether.

**Comment by DatBoiTim** [comment]
Add some way to protect against it. Relegate blood boil as a way to ward off unprepared attackers/random discovery. It should otherwise keep its damage going this route.

Holy water in system/being chaplain(because chaplain being immune innately makes sense) would be the way to go. Since:

A) It allows for prepared raids to ignore it with an item they'll already have ample access to
B) Discourage random vigilantes(read: valid hungry assistants) from raiding without working with chaplain/sec and helping prevent accidental discovery
C) Availability can be mitigated by killing chaplain/destroying the tanks


Edit: Nevermind. Drinking holywater makes you immune. Keep this in and prepare better.

**Comment by Fronsis** [comment]
This looks like an ided moment

as usual improve don't remove

**Comment by Chirynnn** [comment]
> This looks like an ided moment
> 
> as usual improve don't remove

Yes, totally. Absolutely. I am the kind of person who does that because they died to it weeks/months ago and I still am salty about it somehow. Take it as you will. If you read the comments for a moment you'd see some actual better reasons as to why it should (or should not be) removed, but hey what would I know?

**Comment by evermore9932** [comment]
> > This looks like an ided moment
> > as usual improve don't remove
> 
> Yes, totally. Absolutely. I am the kind of person who does that because they died to it weeks/months ago and I still am salty about it somehow. Take it as you will. [...]

You were given plenty of suggestions, all of which are better than shitpr-ing yet another content removal. But that would take some actual effort to code, wouldnt it?

**Review by PowerfulBacon — COMMENTED** [review]
Could the damage either be lowered to make it less overpowered, or a reason provided as to why it should be removed entirely?
I do like the idea of runes that can only be used defensively rather than the usual meta offensive items like swords and shields.
There should also be some dshow that shows the person taking damage that it is attacking them

**Comment by DerpiestBruh** [comment]
> meta offensive items.

Bacon please blood cultists can only summon few weapons

**Comment by PowerfulBacon** [comment]
They only need a few weapons

**Comment by ivanmixo** [comment]
I kinda listed reasons why it should be removed but yeah. "Improve don't remove" a thing that should have never been in the game in the first place.

**Comment by ivanmixo** [comment]
Nothing is stopping you from drawing this in the main halls with 3 pals (or a plushie, lol) and killing anyone who comes by with little effort.

**Comment by PowerfulBacon** [comment]
Whats the problem with lowering the damage or changing the effect slightly?

**Comment by ivanmixo** [comment]
The fact cult can lock down an area by drawing a rune and having 2 friends/1 friend and a plushie with them? Even if you lower the damage that's a very significant advantage over any potential enemy the cult has, not to mention it working on any number of people since the requirement of it applying the damage is so minimal. You can draw it outside brig windows with 2 friends and crit all of sec before they even know what's going on.

Punishing people for not having holy water in their system 24/7 on cult rounds is not the way to go about balance in my opinion. 

Edit: oh also I want to add that it currently does 75 burn and 75 brute. That adds up to 150 damage. Balans.

**Comment by DatBoiTim** [comment]
> The fact cult can lock down an area by drawing a rune and having 2 friends/1 friend and a plushie with them? Even if you lower the damage that's a very significant advantage over any potential enemy the cult has, not to mention it working on any number of people since the requirement of it applying the damage is so minimal. You can draw it outside brig windows with 2 friends and crit all of sec before they even know what's going on.
> 
> Punishing people for not having holy water in their system 24/7 on cult rounds is not the way to go about balance in my opinion. 
> 
> Edit: oh also I want to add that it currently does 75 burn and 75 brute. That adds up to 150 damage. Balans. 


That's such a terrible move if cult does that and wins they deserve it

**Comment by DerpiestBruh** [comment]
Who can tell such horrifying balance can be fixed with a few digit changes

**Comment by Isy232** [comment]
As bacon said, lower the damage.

I'd also add maybe put in some kind of damage effect to the cultists using the rune itself equal to about 1/3rd to 1/5th the dealt damage to prevent spam.

**Comment by PowerfulBacon** [comment]
The damage should be lowered to more acceptible levels. The problem you describe is with the thing being too strong, rather than something being inherently strong with the item. Closing as I don't think this is the correct approach.

**Pull request closed without merging**